### DISCUSSION TITLE
Generate onnx from time series

### DISCUSSION THREAD

**Discussion by agrapps** [autogluon/autogluon#3967] [Q&A] [answered]
Hi, is there a way to convert a model generated from time series (TimeSeriesDataFrame) to onnx, for example i already generate diferent pkl, so is there a way that from that pkl to generate onnx?

My generated models are in: 
autogluon-models > models (Here i have all my models for example AutoARIMA, NPTS, etc) each one with model.pkl

I see this function "MultiModalPredictor.export_onnx" but seems that is only for MultiModalPredictor models

Thanks

**Comment by shchur** [comment, marked as answer]
Hi, currently many models used by `TimeSeriesPredictor` are not implemented in PyTorch (e.g., AutoARIMA is implemented in StatsForecast/Numba and not PyTorch), which is why we cannot provide this functionality.

As for deep learning models that could theoretically support ONNX (e.g., DeepAR, TFT), we'd need to wait for [GluonTS](https://github.com/awslabs/gluonts) to expose this functionality as the provider of these models.

**Reply by agrapps** [reply]
Thanks shchur, hope they realease this functionality :)

Best regards
